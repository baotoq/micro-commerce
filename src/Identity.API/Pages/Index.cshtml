@page
@model IndexModel
@{
    ViewData["Title"] = "Home page";
}

<div class="text-center">
    <h1 class="display-4">Welcome</h1>
    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
    <h2><a href="/.well-known/openid-configuration">openid-configuration</a></h2>
</div>
