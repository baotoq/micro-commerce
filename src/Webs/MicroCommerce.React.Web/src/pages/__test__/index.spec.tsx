import React from "react";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";

import Index from "../index";

describe("Index", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
