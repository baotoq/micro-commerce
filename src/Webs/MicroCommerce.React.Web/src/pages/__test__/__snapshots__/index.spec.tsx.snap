// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should match snapshot 1`] = `
<div>
  <a
    href="/products"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    this page!
  </a>
</div>
`;
